src/Chainweb/Mempool/InMem.hs

```diff
@@ -265,7 +265,7 @@ addToBadListInMem lock txs = withMVarMasked lock $ \mdata -> do
     let !pnd' = foldl' (flip HashMap.delete) pnd txs
     -- we don't have the expiry time here, so just use maxTTL
     now <- getCurrentTimeIntegral
-    let (ParsedInteger mt) = maxTTL
+    let (ParsedInteger mt) = defaultMaxTTL
```

Comment:
Sorry for the lazy question: does the existence of a `defaultMaxTTL` mean that there is some machinery for overriding that default? If so, should that configurable value be used here?

Or, if there are ideas to make it configurable later, and that task will involve grepping for all uses of the default, replacing them with the configured value?

Reply:
I grepped around and actually, i think `InMem.hs` is the only place that actually uses it, so perhaps we might want to just inline it there

Reply:
So to answer your question - there's no means of overriding it once it's in, but it does also apply to transactions we use to test, since we specify a max admissible ttl there too and those can be overwritten.